scipy/stats/_new_distributions.py

```diff
@@ -29,10 +35,23 @@ class Normal(ContinuousDistribution):
     _sigma_domain = _RealInterval(endpoints=(0, inf))
     _x_support = _RealInterval(endpoints=(-inf, inf))
 
-    _mu_param = _RealParameter('mu',  symbol=r'\mu', domain=_mu_domain,
-                               typical=(-1, 1))
-    _sigma_param = _RealParameter('sigma', symbol=r'\sigma', domain=_sigma_domain,
-                                  typical=(0.5, 1.5))
+    _mu_param = _RealParameter(
```

Comment:
It's great that this information appears at the links now! Since these are specified as parameters in the class initializer and available as attributes, can they appear at the front of the lists of Parameters and Attributes? It looks like I could implement it in `_combine_docs` if need be.

Reply:
Hmm good question. I can try and take a look this weekend. If you get to it first, feel free to commit here!